src/library_browser.js

```diff
@@ -292,7 +292,8 @@ var LibraryBrowser = {
           ctx = GL.getContext(contextHandle).GLctx;
         }
         // Set the background of the WebGL canvas to black
-        canvas.style.backgroundColor = "black";
+        if (!contextAttributes["alpha"])
+          canvas.style.backgroundColor = "black";
```

Comment:
This is a line that I have disliked for a very long time. I don't like that we change the style for the user in our runtime like we do here. @kripken : What do you think if we removed this altogether, and instead set the default shells to have a preset black background color on their canvases? That way we could just nuke the attribute setting here for good.